docs/content/integrations/bigquery.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "BigQuery + Dagster"
```

Comment:
```suggestion
title: "Google BigQuery + Dagster"
```

The first time a service/product is mentioned, the provider should be included. Any reference after that can just be the name of the product.